src/functions/Output.ps1

```diff
@@ -698,7 +702,7 @@ function Get-WriteScreenPlugin ($Verbosity) {
         }
 
         foreach ($e in $Context.Block.ErrorRecord) { ConvertTo-FailureLines $e }
-        & $SafeCommands['Write-Host'] -ForegroundColor Red "[-] $($Context.Block.FrameworkData.CommandUsed) $($Context.Block.Path -join ".") failed"
+        & $SafeCommands['Write-Host'] -ForegroundColor $ReportTheme.TestFail "[-] $($Context.Block.FrameworkData.CommandUsed) $($Context.Block.Path -join ".") failed"
```

Comment:
This is a failed block not a failed test. This would happen when you have failure in AfterAll teardown for example.

Reply:
renamed to BlockFail